From 4560648aae6779f43c2a72e6a3fa158e72129a06 Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Fri, 23 Oct 2015 14:26:46 -0400
Subject: [PATCH 0631/1936] Add falktx's DPF

This code will be later used for creating plugin versions.
---
 .gitmodules | 3 +++
 DPF         | 1 +
 2 files changed, 4 insertions(+)
 create mode 160000 DPF

diff --git a/.gitmodules b/.gitmodules
index c582dbb7..677dd880 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -4,3 +4,6 @@
 [submodule "rtosc"]
 	path = rtosc
 	url = https://github.com/fundamental/rtosc
+[submodule "DPF"]
+	path = DPF
+	url = git://github.com/DISTRHO/DPF
diff --git a/DPF b/DPF
new file mode 160000
index 00000000..34d75db7
--- /dev/null
+++ b/DPF
@@ -0,0 +1 @@
+Subproject commit 34d75db7f01fa6b3fb95a6620bf9f6452ad704ea
-- 
2.47.0

